packages/ckeditor5-dev-ci/lib/index.js

```diff
@@ -3,9 +3,7 @@
  * For licensing, see LICENSE.md.
  */
 
-'use strict';
-
-module.exports = {
+export default {
 	getJobApprover: require( './utils/get-job-approver' ),
 	members: require( './data/members.json' )
```

Comment:
This needs to be changed. See: https://github.com/ckeditor/ckeditor5-dev/blob/ci/3759-release-tools/packages/ckeditor5-dev-release-tools/lib/index.js.

For the JSON file, we have the options `fs-extra` and `readJson()`, or we can use `import with`. Whatever we decide, we must be consistent across the repository.

@ckeditor/ckeditor-5-devops, WDYT?

Reply:
If it is possible, let's use native solutions wherever possible.